tools/cd_scripts/e2e_test.sh

```diff
@@ -58,8 +58,8 @@ then
     sudo apt install -y fuse
 
     # download and install gcsfuse deb package
-    gsutil cp gs://gcsfuse-release-packages/v$(sed -n 1p details.txt)/gcsfuse_$(sed -n 1p details.txt)_amd64.deb .
-    sudo dpkg -i gcsfuse_$(sed -n 1p details.txt)_amd64.deb |& tee -a ~/logs.txt
+    gsutil cp gs://gcsfuse-release-packages/v$(sed -n 1p details.txt)/gcsfuse_$(sed -n 1p details.txt)_$(dpkg --print-architecture).deb .
```

Comment:
define and use variables for repeated commands like $(dpkg --print-architecture) & $(uname -i)